webview/src/plots/components/comparisonTable/DropTargetIndicator.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { DragEnterDirection } from '../../../shared/components/dragDrop/util'
+import styles from '../styles.module.scss'
+
+export const DropTargetIndicator: React.FC<Props> = ({ direction }) => (
+  <span
+    className={styles.dropTargetIndicator}
+    data-testid="comparison-drop-target-indicator"
+    style={{
+      [direction.toLowerCase()]: -4
```

Comment:
Using the transformed value of an enum directly as a string makes me a little uneasy, though it's not like avoiding doing so makes the code that much more elegant.

Reply:
Something I've seen done in Elm on cases like this, since in Elm you can't derive a string from an enum, is a function that explicitly returns a string from an enum value. I thought about doing the same thing here, but it gave me "early optimization" vibes. Does team have any ideas or precedents in the code base for tackling this?